app/src/main/java/com/willowtree/vocable/presets/AddToCategoryPickerFragment.kt

```diff
@@ -133,6 +133,8 @@ class AddToCategoryPickerFragment : BaseFragment<FragmentAddToCategoryPickerBind
                     )
                 }
             }
+        } else {
+            binding.categoryPageNumber.text = getString(R.string.phrases_page_number, 1, 1)
```

Comment:
I don't know what R is, but LGTM ;)

Reply:
R is the parent for all of Android's resource files - strings, integers, dimens, colors, etc :)

Reply:
Nice! Now I know! :)